AnalyticsRallyHere/Source/AnalyticsRallyHere/Private/RH_AnalyticsProviderEventCache.cpp

```diff
@@ -0,0 +1,543 @@
+// Copyright 2022-2023 Rally Here Interactive, Inc. All Rights Reserved.
+
+#include "RH_AnalyticsProviderEventCache.h"
+#include "Analytics.h"
+#include "Misc/ScopeLock.h"
+#include "PlatformHttp.h"
+#include "Algo/Accumulate.h"
+#include "Serialization/JsonWriter.h"
+#include "Containers/StringConv.h"
+#include "Misc/StringBuilder.h"
+#include "Misc/CString.h"
+#include "HAL/IConsoleManager.h"
+#include "HAL/PlatformTime.h"
+
+namespace EventCacheStatic
+{
+	static float PayloadPercentageOfMaxForWarning = 1.00f;
+	FAutoConsoleVariableRef CvarPayloadPercentageOfMaxForWarning(
+		TEXT("AnalyticsET.PayloadPercentageOfMaxForWarning"),
```

Comment:
This references AnalyticsET

Reply:
Fixed this and one other spot still referencing AnalyticsET